powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/adapter/OperationCustomizationService.java

```diff
@@ -40,6 +49,7 @@
     private final Logger logger = LoggerFactory.getLogger(OperationCustomizationService.class);
 
     private final OperationConverter operationConverter = new OperationConverter();
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Use autowired instance?

Reply:
I'll review the object mapper initialization code, this will be on many places, I think.